provisioning/ubuntu-provision.sh

```diff
@@ -348,6 +348,14 @@ function install_docker_compose(){
   chmod a+x /usr/local/bin/docker-compose
 }
 
+## Ensure that DOCTL is installed
+function install_doctl(){
+  curl --fail --silent --location --show-error --output /usr/local/bin/doctl \
+    "https://github.com/digitalocean/doctl/releases/download/v${DOCTL_VERSION}/doctl-${DOCTL_VERSION}-linux-$(uname -m).tar.gz" | tar -xzv doctl
```

Comment:
This command is invalid:

- `curl` downloads the tar gz archive to `/usr/local/bin/doctl`
- There is a `tar` command piped from the `curl` which does nothing (as there are no output on the stdout from `curl`)